ports/xbyak/portfile.cmake

```
@@ -3,23 +3,19 @@ vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO herumi/xbyak
-    REF 77ffe717376d194f1e5cc910bbd755d99bbba94e	# 5.991 + cmake targets
-    SHA512 4e1ffa98dc63444aebf0a6e7a4ea57a9040cbf261f3a6fa43d23f0361cece71f1f4b597e95b6c85efbd4668e421081730b04ee924450ea0b0114923c36fc9c74
+    REF v6.00
+    SHA512 8bea95ddb44be39312083173cd4401039bd402c4732d97f9ff63174f6db29793f80dc1f1589d64bd96b4b13af819133897cd9843523662d13d57158c3837a778
     HEAD_REF master
 )
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
-)
-
-vcpkg_install_cmake()
+vcpkg_cmake_configure(SOURCE_PATH "${SOURCE_PATH}")
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/${PORT})
+vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/${PORT}")

```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/${PORT})
```
Quotes are not needed for a relative path.

---

The [maintainer cmake guidelines](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/cmake-guidelines.md#the-guidelines) state the following in regards to arguments with variable expansion:
* If an argument contains a variable expansion `${...}`, it must be quoted.

---

Yes, you're right. For most `${...}`, quotes are required. But for `${PORT}`. it doesn't refer to a path. 
Through internal discussion, adding or not are both okay for this case. please see https://github.com/microsoft/vcpkg/pull/20324#discussion_r716445250